iOS Client/PodcastsController/PodcastDetailViewController.swift

```diff
@@ -79,17 +117,27 @@ class PodcastDetailViewController: UIViewController, StoreSubscriber {
     
     func newState(state: MBAppState) {
         switch state.podcastsState.player {
-        case .error, .initialized, .paused, .finished:
-            playPauseButton.setImage(#imageLiteral(resourceName: "play"), for: .normal)
+        case .initialized, .error, .paused, .finished:
+            playPauseButton.setImage(#imageLiteral(resourceName: "play-arrow"), for: .normal)
+            if playerState == .playing, playerState != .initialized { animateImage(size: CGSize(width: 150, height: 150), duration: 0.6)}
         case .playing:
-            playPauseButton.setImage(#imageLiteral(resourceName: "pause"), for: .normal)
+            playPauseButton.setImage(#imageLiteral(resourceName: "pause-bars"), for: .normal)
+            if playerState != .playing, playerState != .initialized { animateImage(size: CGSize(width: 200, height: 200), duration: 0.6)}
```

Comment:
why the check for initialized? It seems to me you're trying to catch toggles between playing and not playing to do some image animation

Going from `initialized` to `playing` seems like a good time to blow up the image. Btw, constants might be good here too. They can live in this file at the top. That'll put config in one place

Reply:
Can't remember exactly why I check for initialized here. I think it has to do with when the view appears it looks funky if it is animating while pushing onto the screen.

Reply:
yeah that is why ^^^

Reply:
ok cool